examples/java/plugin-auto-refresh/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM openjdk:8u212-jdk-alpine as builder
+
+RUN apk --no-cache add curl \
+    && addgroup -S app && adduser -S -g app app \
+    && mkdir -p /opt/ && cd /opt/ \
+    && echo "Downloading gradle.." \
+    && curl -sSfL https://services.gradle.org/distributions/gradle-5.5.1-bin.zip -o gradle-bin.zip \
+    && unzip gradle-bin.zip -d /opt/ \
+    && rm gradle-bin.zip \
+    && apk del curl
+
+# Export some environment variables
+
+FROM openfaas/of-watchdog:0.7.6 as watchdog
+
+FROM openjdk:8u212-jdk-alpine as ship
```

Comment:
Can we use v11 again please?

Reply:
@embano1 I don't think this is possible at the moment for SDK & plugins reasons, not OpenFaaS ones. The SDK sample plugin is Java8 based. Also this function is not using any OpenFaaS template, for OpenFaaS it is just an endpoint - behind the scenes it can be any language and any version.